docs/openhabian-amanda.md

```diff
@@ -567,12 +567,22 @@ You could also move that temporary recovered image file to your Windows PC that
 ### A final word on when things have gone badly wrong...
 
 and your SD card to contain the Amanda database is broken: you don't have to give up.
-Whenever you use a directory as the storage area, openHABian Amanda by default creates a copy of its config and index files (to know what's stored where) in your storage directory once a day (see `/etc/cron.d/amanda`). This is usually the amanda-backups directory, rather than the slots directory, and you extract the /var/lib/amanda files from the amanda_data_DATE.tar.gz file you want (probably the latest). You can also extract the /var/log/amanda directory if needed.
+Whenever you use a directory as the storage area, openHABian Amanda by default creates a copy of its config and index files (to know what's stored where) in your storage directory once a day (see `/etc/cron.d/amanda`). You need to gain an understanding of how your physical storage device works, whether it is a USB stick, USB drive, NAS device on your network, or other device. For example, using a web browser view of a NAS device you will find an amanda-backups directory and a slots directory under the folder you assigned for amanda to use on the NAS.
+
+Example files are
+/volume1/rp3Backup/amanda-backups/amanda_data_20200331020001.tar.gz
+/volume1/rp3Backup/slots/slot14/00000.openHABian-openhab-dir-014
+/volume1/rp3Backup/slots/slot14/00001.openhab._var_lib_openhab2.1
+/volume1/rp3Backup/slots/slot14/00002.openhab._etc_openhab2.0
+
```

Comment:
Apply proper formatting (backticks for codestyle font, no italics).

See also your other additions.